ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

```diff
@@ -8,8 +8,7 @@ Resource       ./Workbenches.resource
 ${DS_PROJECT_TITLE}=    Data Science Projects
 ${TITLE_INPUT_XP}=    xpath=//input[@id="manage-project-modal-name"]
 ${DESCR_INPUT_XP}=    xpath=//textarea[@id="manage-project-modal-description"]
-${RESOURCE_EDIT_BTN_XP}=    xpath=//button[.="Edit resource name"]
-${RESOURCE_INPUT_XP}=    css:[data-testid="manage-project-modal-resourceName"]
+${RESOURCE_INPUT_XP}=     css:[data-testid="resource-manage-project-modal-name"]
```

Comment:
waits for https://github.com/red-hat-data-services/ods-ci/pull/1866 to be merged